cmd/thv-operator/api/v1alpha1/virtualmcpserver_types.go

```diff
@@ -53,6 +55,11 @@ type VirtualMCPServerSpec struct {
 	// +kubebuilder:pruning:PreserveUnknownFields
 	// +kubebuilder:validation:Type=object
 	PodTemplateSpec *runtime.RawExtension `json:"podTemplateSpec,omitempty"`
+
+	// Telemetry configures OpenTelemetry-based observability for the Virtual MCP server
+	// including distributed tracing, OTLP metrics export, and Prometheus metrics endpoint
+	// +optional
+	Telemetry *telemetry.Config `json:"telemetry,omitempty"`
```

Comment:
hmm, I don't see the deepcopy files or the CRDs being regenerated, did you run:
```
  task operator-generate
  task operator-manifests
  cd cmd/thv-operator && task crdref-gen
```
?